Test.C

```diff
@@ -53,9 +49,11 @@ struct Round {
     Attack attack;
 };
 
-int main(){
+//Function prototypes
+void briefing(Player *player);
```

Comment:
```suggestion
void displayBriefingMessage(Player *playerPtr);
```
It's probably best practice to use nouns as variable names and verbs as function names. Also, if we are going to put parameter names in the prototype, they should match the names in the function declaration. Another option for this could be eliminating the parameter name in the prototype: `displayBriefingMessage(Player *);`

Reply:
Good catch, and I totally agree with verbs for functions.